pkgs/top-level/all-packages.nix

```diff
@@ -14254,6 +14254,8 @@ with pkgs;
 
   clickable = python3Packages.callPackage ../development/tools/clickable { };
 
+  ubports-click = python3Packages.callPackage ../development/tools/click { };
```

Comment:
Nit: Not alphabetically ordered anymore (although the order in that file is already kinda all over the place… eeh)